data/rsp.rodata.s

```diff
@@ -33,5 +33,5 @@ glabel D_80157580
 glabel D_801579A0
     .incbin "baserom.z64", 0xBCEB40, 0x390
 
-glabel D_80157D30
+glabel njpegUCodeData
```

Comment:
we only really use the `g` prefix for global, and `s` for static. Both of these youve changed here look like they should be g.
rest looks good to me

Reply:
"njpeg" is a term here, referring to Nintendo's JPEG decoding microcode. What do you think, `gNjpeg` or `gNJpeg`?

Reply:
is it an existing convention to call it nJpeg, or is that something you added now on your own?
If the latter, I dont see a need to specify that its nintendos personally. Everything in this codebase already is

Reply:
There's a `#define M_NJPEGTASK 4` I based it on (include/ultra64/sptask.h)

Reply:
hmm ok. I think we can probably take our liberties with the names here, but it doesnt matter too much to me atm. Can leave the n for now and figure out later if we want to keep 

but yea, `gNJpeg..` would be the way to go if we're keeping it

Reply:
I think I've been convinced, for two reasons - 1. as you said, of course it's Nintendo, and 2. "UCode" already makes it pretty clear anyway